vuepress/docs/next/docs/create/component-gen-overview.md

```diff
@@ -1,42 +1,44 @@
+- Customize as desired: The generated code is ready to be developed on and modified to enable features and capabilities.
+- Avoid a learning curve: The Entando Blueprint uses standard tools, technologies, and patterns that can be consumed by developers who are not familiar with the underlying technology.
+
+### Standardize Technology 
+
+- Guarantee technology compliance: The Component Generator encodes business standards for networking, monitoring, build tools, and distribution into the Entando Blueprint to ensure that business requirements are satisfied.
+- Guarantee internal compliance: The Entando Blueprint provides parameters to ensure the code complies with standards and practices of your organization including formatting, testing tools, static analysis tools, and package structures.
+- Jumpstart 3rd party adoption: Teams can quickly recognize and adopt code from system integrators and other third parties because these patterns and standards are encoded into the default application.
 
 ## Generated Component Flow
 
-This section provides an overview of the component generation process. If you want to jump into the code and a tutorial start here: [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)
+The following provides an overview of the component generation process. To view the code and learn how to use the Entando Blueprint, refer to the [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md).
 
 1. Create components
-   - The Component Generator will create a Spring Boot microservice and optionally will create React micro frontends for entities that you add to your project
-   - The microservice and generated micro frontends are runnable out of the box and preconfigured to connect to [Entando Identity Management](../consume/identity-management.md)
-   - The code is meant to be edited and enhanced by developers. JHipster provides interactive updates and merges if you need to re-run the generator or add new functionality via JHipster
-   - The microservice includes liquibase and integrated data mocking if you want to generate data in your API calls
+   - The Component Generator creates a Spring Boot microservice and, optionally, React micro frontends for entities that you add to your project.
+   - The microservice and generated micro frontends are out-of-the-box runnable and preconfigured to connect to the [Entando Identity Management System](../consume/identity-management-system.md).
+   - To integrate user edits and enhancements to the code, JHipster offers interactive updates and merges to re-run the modified generator. Alternatively, the user can add new functionality via JHipster.
+   - The microservice includes Liquibase and integrated mocking to offer data generation in API calls.
```

Comment:
would keep 'data' in place, unless integrated mocking is a 'thing'. looking it up, seems like mocking can apply to different services, and here it's specifically about data.

Reply:
i don't think any value is lost with removing "data"; the sentence makes it clear that the thing you're handling is data. that being said, no harm in being too clear...